test/Tester/LogFomatterTests.cs

```diff
@@ -284,6 +372,17 @@ private TestLoggerFactory BuildNamedOptionsExpectedResult()
             return expected;
         }
 
+        private class TestOptionsWithSecrets
```

Comment:
suggest including a wider variety of property types, including things like ints and lists to see what happens.

Reply:
string are already tested, not sure if it is really necessary to test other types, as long as they have a string representation...

For enumerable type that's an interesting question. For now I left them as is, since in our codebase sometimes we concatenate a full list to a string, sometimes we do something else...